redash/controllers.py

```diff
@@ -450,7 +450,7 @@ def delete(self, visualization_id):
 class QueryResultListAPI(BaseResource):
     @require_permission('execute_query')
     def post(self):
-        params = request.json
```

Comment:
:-) true that.

I wonder if once I get to fix the test helper, we should keep the `get_json(force=True)` (it's used in other places too, I think) or not ...